src/ui/app.js

```diff
@@ -340,8 +341,9 @@ class ArchiveWebApp extends ReplayWebApp
     `);
   }
 
+  // HACK: returns the logo requested by ReplayWeb.page's nav as nothing now that the new logo includes both graphics and text. Probably best to refactor this behavior.
   get mainLogo() {
-    return wrLogo;
```

Comment:
yeah, this is probably something we can cleanup as part of AWP typescript conversion, this is probably best option for now.